### PR TITLE
fix(ingestion/looker): fix doc for sql parsing documentation

### PR DESCRIPTION
**Pull request by sid-acryl**
## Summary by CodeRabbit

- **Documentation**
  - Simplified and rephrased configuration notes regarding SQL parsing.
  - Added a note about setting liquid variables for views containing liquid templates.

### CODE REVIEW

File: metadata-ingestion/docs/sources/looker/lookml_post.md [hunk 1]
```diff
@@ -1,14 +1,7 @@
...
-:::note
-
-The integration can use an SQL parser to try to parse the tables the views depends on.
-
-:::
-
-This parsing is disabled by default, but can be enabled by setting `parse_table_names_from_sql: True`. The default parser is based on the [`sqllineage`](https://pypi.org/project/sqllineage/) package.
-As this package doesn't officially support all the SQL dialects that Looker supports, the result might not be correct. You can, however, implement a custom parser and take it into use by setting the `sql_parser` configuration value. A custom SQL parser must inherit from `datahub.utilities.sql_parser.SQLParser`
-and must be made available to Datahub by ,for example, installing it. The configuration then needs to be set to `module_name.ClassName` of the parser.
+1.  The sql parsing is disabled by default, but can be enabled by setting `parse_table_names_from_sql: True`.
```
Comment (hsheth2) [inline metadata-ingestion/docs/sources/looker/lookml_post.md:3]: should we enable this by default?

File: metadata-ingestion/docs/sources/looker/lookml_post.md [hunk 2]
```diff
@@ -1,14 +1,7 @@
...
+2.  If view contains liquid template then we can set liquid variable as key value pair in `liquid_variable` config parameter where key act as variable-name and value as variable's value
```
Comment (hsheth2) [inline metadata-ingestion/docs/sources/looker/lookml_post.md:4]: link to the looker docs on Liquid templates and provide an example of some lkml and the corresponding `liquid_variable` config

```suggestion
2.  If a view contains a liquid template (e.g. `{{ user_attributes['whatever'] }}`), then you will need to specify the values of those variables in the `liquid_variable` config
```

File: metadata-ingestion/docs/sources/looker/lookml_post.md (hunk 1 above)
Comment (sid-acryl) [reply metadata-ingestion/docs/sources/looker/lookml_post.md:3]: It is done as part of https://github.com/datahub-project/datahub/pull/10892

Comment (hsheth2) [reply metadata-ingestion/docs/sources/looker/lookml_post.md:3]: we can remove this line then right

File: metadata-ingestion/docs/sources/looker/lookml_post.md [hunk 3]
```diff
@@ -1,14 +1,7 @@
...
-
-The integration can use an SQL parser to try to parse the tables the views depends on.
-
-:::
-
-This parsing is disabled by default, but can be enabled by setting `parse_table_names_from_sql: True`. The default parser is based on the [`sqllineage`](https://pypi.org/project/sqllineage/) package.
-As this package doesn't officially support all the SQL dialects that Looker supports, the result might not be correct. You can, however, implement a custom parser and take it into use by setting the `sql_parser` configuration value. A custom SQL parser must inherit from `datahub.utilities.sql_parser.SQLParser`
-and must be made available to Datahub by ,for example, installing it. The configuration then needs to be set to `module_name.ClassName` of the parser.
+1.  The sql parsing is disabled by default, but can be enabled by setting `parse_table_names_from_sql: True`.
+2.  If a view contains a liquid template (e.g. `{{ user_attributes['whatever'] }}`), then you will need to specify the values of those variables in the `liquid_variable` config
```
Comment (hsheth2) [inline metadata-ingestion/docs/sources/looker/lookml_post.md:4]: actually provide an example lookml code and corresponding recipe

also make it a bit more realistic - people might have something like this `sql_table_name: {{ user_attributes['db']}}.kafka_streaming.events`, with `db=ANALYTICS_PROD`

File: metadata-ingestion/docs/sources/looker/lookml_post.md [hunk 4]
```diff
@@ -1,14 +1,12 @@
 #### Configuration Notes
 
-:::note
-
-The integration can use an SQL parser to try to parse the tables the views depends on.
-
-:::
+1.  If a view contains a liquid template (e.g. `sql_table_name: {{ user_attributes['db']}}.kafka_streaming.events, with db=ANALYTICS_PROD`), then you will need to specify the values of those variables in the `liquid_variable` config as shown below
```
Comment (hsheth2) [inline metadata-ingestion/docs/sources/looker/lookml_post.md:3]:
```suggestion
1. If a view contains a liquid template (e.g. `sql_table_name: {{ user_attributes['db']}}.kafka_streaming.events }}`, with `db=ANALYTICS_PROD`), then you will need to specify the values of those variables in the `liquid_variable` config as shown below:
```

File: metadata-ingestion/docs/sources/looker/lookml_post.md [hunk 5]
```diff
@@ -1,14 +1,12 @@
 #### Configuration Notes
 
-:::note
-
-The integration can use an SQL parser to try to parse the tables the views depends on.
-
-:::
+1.  If a view contains a liquid template (e.g. `sql_table_name: {{ user_attributes['db']}}.kafka_streaming.events, with db=ANALYTICS_PROD`), then you will need to specify the values of those variables in the `liquid_variable` config as shown below 
+    ```
```
Comment (hsheth2) [inline metadata-ingestion/docs/sources/looker/lookml_post.md:4]: yml

File: metadata-ingestion/docs/sources/looker/lookml_post.md [hunk 6]
```diff
@@ -1,14 +1,12 @@
...
+      liquid_variable:
+        user_attributes:
+          db: ANALYTICS_PROD
```
Comment (hsheth2) [inline metadata-ingestion/docs/sources/looker/lookml_post.md:8]:
```suggestion
```

**Review by hsheth2 — APPROVED** [review]
Please pay more attention to the formatting details

You can see a preview of the change via the "deployment" link posted by Vercel: https://docs-website-bun5cd5kr-acryldata.vercel.app/docs/generated/ingestion/sources/looker/#config-details-1

File: metadata-ingestion/docs/sources/looker/lookml_post.md (hunk 5 above)
Comment (hsheth2) [inline metadata-ingestion/docs/sources/looker/lookml_post.md:4]:
```suggestion
    ```yml
```

**Pull request merged**